doc/manual/manual.tex

```diff
@@ -296,33 +296,24 @@ \subsection{Referencing \aspect{}}
   journal =      {Geophysics Journal International},
   year =         2012,
   volume =       191,
-  pages =        {12--29}}
+  pages =        {12--29}
+}
 \end{lstlisting}
-
-and cite the User's Manual:
+Please also cite the User's Manual:
 \begin{lstlisting}[frame=single,language=tex]
 @Manual{aspectmanual,
-        title =        {\textsc{ASPECT}: Advanced Solver for Problems in Earth's
-                        ConvecTion v1.5.0}, 
-        author =       {W. Bangerth and J. Dannberg and 
-                        R. Gassm{\"o}ller and T. Heister and others},
-        organization = {Computational Infrastructure for Geodynamics},
-        year =         2016
-      }
-\end{lstlisting}
-
-To help your reader find the \aspect{} software package, you may also refer to the website by its url or by citing the following:
-\begin{lstlisting}[frame=single,language=tex]
-@Manual{aspectweb,
-        title  = {ASPECT: Advanced Solver for Problems in Earth's ConvecTion},
-        author = {W. Bangerth and J. Dannberg and 
+  title =        {\textsc{ASPECT}: Advanced Solver for Problems in Earth's
+                  ConvecTion v1.5.0}, 
```

Comment:
In fact, all of the capitalized words have to be adjusted. Will open different issue for this.